MediaGalleryUi/view/adminhtml/web/template/image-uploader.html

```diff
@@ -11,8 +11,7 @@
     </form>
     <div data-role="spinner" class="admin__data-grid-loading-mask" data-bind="visible: loader">
         <div class="spinner">
-            <span></span><span></span><span></span><span></span>
-            <span></span><span></span><span></span><span></span>
+            <span repeat="8"/>
```

Comment:
Is it possible to get red of this at all?
Maybe, use styles to properly display spinner?